_posts/ja/releases/2021-05-01-release-0.21.1.md

```diff
@@ -0,0 +1,218 @@
+2021年8月11日を経過した最初のリターゲット期間の開始時よりも前に、
+リターゲット期間2,016ブロック（約2週間）の90%のブロックがTaprootのサポートを通知した場合、
+ソフトフォークはロックインされ、Taprootはブロック709632から有効になります（11月上旬もしくは中旬の予定）。
+
+Speedy TrialアクティベーションでTaprootがロックインされなかった場合は、
+Speedy Trial方式が失敗した理由に対応するための変更を加えた後続のアクティベーション方式がデプロイされると予想されます。
+
+本リリースにはTaprootアドレスへの支払い機能が含まれていますが、
+そのようなアドレスへの支払いはTaprootがアクティベートされるまで安全ではありません。
+また、アクティベーション後にTaprootトランザクションをリレー、マイニングする機能も含まれています。
+本リリースには、この2つの基本機能を除き、誰もがtaprootを直接使用できるようなコードは含まれていません。
+Bitcoin CoreウォレットにTaproot関連の機能が追加されるのは、
+Taprootのアクティベーションが確実になった後のリリースになる見込みです。
+
+すべてのユーザー、企業およびマイナーはTaprootのアクティベーションに異議がない限り、
+このリリース（もしくは後続の互換性のあるリリース）にアップグレードすることが推奨されます。
+Taprootがロックインされた場合、ブロック709632の前にアップグレードし、
+Taprootの新しいルールを適用し、誤って確認されたトランザクションが発生する可能性を回避することを強くお勧めします。
+
+Taprootをアクティベートしたいマイナーは、できればこのリリースを使って通知を制御する必要があります。
+`getblocktemplate`RPCの結果は、適切なスタートになると自動的に通知を更新し、タイムアウトが発生するか、
+Taprootがアクティベートされるまで通知を続けます。
+あるいは、マイナーはいつでも手動でビット2の通知を開始できます。
+Taprootがアクティベートされると、ブロック709632の前にノードを確実に更新する必要があり、
+更新されていないノードでは無効なチェーンがマイニングされる可能性があります。
+詳しくは、[versionbits FAQ](https://bitcoincore.org/en/2016/06/08/version-bits-miners-faq/)をご覧ください。
+
+Taprootの詳細については、以下のリソースを参照ください:
+
+- 技術仕様
+  - [BIP340 Schnorr signatures for secp256k1](https://github.com/bitcoin/bips/blob/master/bip-0340.mediawiki) 
+  - [BIP341 Taproot: SegWit version 1 spending rules](https://github.com/bitcoin/bips/blob/master/bip-0341.mediawiki)
+  - [BIP342 Validation of Taproot scripts](https://github.com/bitcoin/bips/blob/master/bip-0342.mediawiki)
+
+- 人気記事
+  - [Taproot Is Coming: What It Is, and How It Will Benefit Bitcoin](https://bitcoinmagazine.com/technical/taproot-coming-what-it-and-how-it-will-benefit-bitcoin)
+  - [What do Schnorr Signatures Mean for Bitcoin?](https://academy.binance.com/en/articles/what-do-schnorr-signatures-mean-for-bitcoin)
+  - [The Schnorr Signature & Taproot Softfork Proposal](https://blog.bitmex.com/the-schnorr-signature-taproot-softfork-proposal/)
+
+- 開発経緯の概要
+  - [Taproot](https://bitcoinops.org/en/topics/taproot/)
+  - [Schnorr signatures](https://bitcoinops.org/en/topics/schnorr-signatures/)
+  - [Tapscript](https://bitcoinops.org/en/topics/tapscript/)
+  - [Soft fork activation](https://bitcoinops.org/en/topics/soft-fork-activation/)
+
+- その他
+  - [Questions and answers related to taproot](https://bitcoin.stackexchange.com/questions/tagged/taproot)
+  - [Taproot review](https://github.com/ajtowns/taproot-review)
+
+更新されたRPC {#updated-rpcs}
+------------
+
+- [BIP 350](https://github.com/bitcoin/bips/blob/master/bip-0350.mediawiki)が実装されたことで、
+  アドレスを受け取るすべてのRPCでnative witness version 1（またはそれ以上）が渡された場合の動作が変更されました。
+  これらはBech32ではなくBech32mエンコーディングを必要とするようになり、RPCの出力でもそのようなアドレスには
+  Bech32mエンコーディングが使用されます。version 1のアドレスは、
+  それらに意味を与えるコンセンサスルールが採用されるまで
+  （例えば、[BIP 341](https://github.com/bitcoin/bips/blob/master/bip-0341.mediawiki)を介して）、
+  mainnetでは作成しないでください。作成されると、Bech32mが使用されることになるでしょうから、
+  プロダクションシステムには影響ありませんが、
+  （signetのように）そのようなアドレスがすでに意味を持っている他のネットワークで観察される可能性があります。
```

Comment:
Google Translate is making this change sound a lot more exciting than I think was intended; quote (emphasis added):

> Once created, Bech32m will be used, so it won't affect the production system, but it can be observed on other networks where such addresses already make sense (like signet). **There is sex.**

I only get this in the Chrome in-browser translation, not when [pasting directly](https://translate.google.com/?sl=auto&tl=en&text=BIP%20350%E3%81%8C%E5%AE%9F%E8%A3%85%E3%81%95%E3%82%8C%E3%81%9F%E3%81%93%E3%81%A8%E3%81%A7%E3%80%81%0A%20%20%E3%82%A2%E3%83%89%E3%83%AC%E3%82%B9%E3%82%92%E5%8F%97%E3%81%91%E5%8F%96%E3%82%8B%E3%81%99%E3%81%B9%E3%81%A6%E3%81%AERPC%E3%81%A7native%20witness%20version%201%EF%BC%88%E3%81%BE%E3%81%9F%E3%81%AF%E3%81%9D%E3%82%8C%E4%BB%A5%E4%B8%8A%EF%BC%89%E3%81%8C%E6%B8%A1%E3%81%95%E3%82%8C%E3%81%9F%E5%A0%B4%E5%90%88%E3%81%AE%E5%8B%95%E4%BD%9C%E3%81%8C%E5%A4%89%E6%9B%B4%E3%81%95%E3%82%8C%E3%81%BE%E3%81%97%E3%81%9F%E3%80%82%0A%20%20%E3%81%93%E3%82%8C%E3%82%89%E3%81%AFBech32%E3%81%A7%E3%81%AF%E3%81%AA%E3%81%8FBech32m%E3%82%A8%E3%83%B3%E3%82%B3%E3%83%BC%E3%83%87%E3%82%A3%E3%83%B3%E3%82%B0%E3%82%92%E5%BF%85%E8%A6%81%E3%81%A8%E3%81%99%E3%82%8B%E3%82%88%E3%81%86%E3%81%AB%E3%81%AA%E3%82%8A%E3%80%81RPC%E3%81%AE%E5%87%BA%E5%8A%9B%E3%81%A7%E3%82%82%E3%81%9D%E3%81%AE%E3%82%88%E3%81%86%E3%81%AA%E3%82%A2%E3%83%89%E3%83%AC%E3%82%B9%E3%81%AB%E3%81%AF%0A%20%20Bech32m%E3%82%A8%E3%83%B3%E3%82%B3%E3%83%BC%E3%83%87%E3%82%A3%E3%83%B3%E3%82%B0%E3%81%8C%E4%BD%BF%E7%94%A8%E3%81%95%E3%82%8C%E3%81%BE%E3%81%99%E3%80%82version%201%E3%81%AE%E3%82%A2%E3%83%89%E3%83%AC%E3%82%B9%E3%81%AF%E3%80%81%0A%20%20%E3%81%9D%E3%82%8C%E3%82%89%E3%81%AB%E6%84%8F%E5%91%B3%E3%82%92%E4%B8%8E%E3%81%88%E3%82%8B%E3%82%B3%E3%83%B3%E3%82%BB%E3%83%B3%E3%82%B5%E3%82%B9%E3%83%AB%E3%83%BC%E3%83%AB%E3%81%8C%E6%8E%A1%E7%94%A8%E3%81%95%E3%82%8C%E3%82%8B%E3%81%BE%E3%81%A7%0A%20%20%EF%BC%88%E4%BE%8B%E3%81%88%E3%81%B0%E3%80%81BIP%20341%E3%80%81%0A%20%20mainnet%E3%81%A7%E3%81%AF%E4%BD%9C%E6%88%90%E3%81%97%E3%81%AA%E3%81%84%E3%81%A7%E3%81%8F%E3%81%A0%E3%81%95%E3%81%84%E3%80%82%E4%BD%9C%E6%88%90%E3%81%95%E3%82%8C%E3%82%8B%E3%81%A8%E3%80%81Bech32m%E3%81%8C%E4%BD%BF%E7%94%A8%E3%81%95%E3%82%8C%E3%82%8B%E3%81%93%E3%81%A8%E3%81%AB%E3%81%AA%E3%82%8B%E3%81%A7%E3%81%97%E3%82%87%E3%81%86%E3%81%8B%E3%82%89%E3%80%81%0A%20%20%E3%83%97%E3%83%AD%E3%83%80%E3%82%AF%E3%82%B7%E3%83%A7%E3%83%B3%E3%82%B7%E3%82%B9%E3%83%86%E3%83%A0%E3%81%AB%E3%81%AF%E5%BD%B1%E9%9F%BF%E3%81%82%E3%82%8A%E3%81%BE%E3%81%9B%E3%82%93%E3%81%8C%E3%80%81%0A%20%20%EF%BC%88signet%E3%81%AE%E3%82%88%E3%81%86%E3%81%AB%EF%BC%89%E3%81%9D%E3%81%AE%E3%82%88%E3%81%86%E3%81%AA%E3%82%A2%E3%83%89%E3%83%AC%E3%82%B9%E3%81%8C%E3%81%99%E3%81%A7%E3%81%AB%E6%84%8F%E5%91%B3%E3%82%92%E6%8C%81%E3%81%A3%E3%81%A6%E3%81%84%E3%82%8B%E4%BB%96%E3%81%AE%E3%83%8D%E3%83%83%E3%83%88%E3%83%AF%E3%83%BC%E3%82%AF%E3%81%A7%E8%A6%B3%E5%AF%9F%E3%81%95%E3%82%8C%E3%82%8B%E5%8F%AF%E8%83%BD%E6%80%A7%E3%81%8C%E3%81%82%E3%82%8A%E3%81%BE%E3%81%99%E3%80%82&op=translate) into Google Translate, so I'm going to assume this is just a bug in Google.  No changes required here unless you want to make some.

Reply:
The translation will be the same if you remove line breaks.

Reply:
Google Translate seems to have translated `性があります` at the end of the sentence as `There is sex.`. I don't know why Google Translate separated sentences there, but `可能性があります` is a translation of the context of `mey be`.